help/en/html/tools/logixng/reference/chapter5.shtml

```diff
@@ -558,28 +619,53 @@
 
         <li><strong>Set Train Name:</strong> Set the train Name for the selected Warrant.</li>
       </ul>
-      <a id="digital_action_common"></a>
-      <h3>Digital action :: Common</h3>
+
+      <!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
+
+      <h2 id="digital_action_common">Digital action :: Common</h2>
 
       <div style="margin-left: 2em;">
         <a href="images/chapter5/digital_action_common.png"><img src=
         "images/chapter5/digital_action_common.png" alt="Chapter 5 digital action common list"
-        width="204" height="170"></a>
+        width="204" height="188"></a>
+      </div>
+
+      <!-- = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = = -->
+
+      <h4>Digital Formula<span class="since">since 4.25.8</span></h4>
+
+      <p>The <strong>Digital Formula</strong> action provides the ability to create formulas that
+      are not directly related to other items, such as sensors, which use the Formula tab.</p>
```

Comment:
Maybe add something like:

The formula action is in particular useful when working with local variables. You can for example increase a local variable by the formula `index++` or make some calculations, like `turnout = name.substring(2)` which removes the first two characters from the string.

Reply:
Done.